apps/native-component-list/package.json

```diff
@@ -47,6 +47,7 @@
     "@react-navigation/drawer": "^7.3.9",
     "@react-navigation/elements": "^2.3.8",
     "@react-navigation/native": "^7.1.6",
+    "@react-navigation/native-stack": "^7.3.21",
```

Comment:
Testing in bare-expo was enough but we'd like to avoid adding native-stack to `native-component-list` as it's used in many places within different apps. Could you revert the changes made to NCL (package and native navigator) before merging this PR?

Reply:
Sorry for the confusion 😄  I meant to remove the changes made in `apps/native-component-list`

Reply:
Done!

Reply:
Thank you!